back-end/routes/user/index.js

```diff
@@ -43,4 +43,49 @@ router.get("/:name", async (req, res, next) => {
   }
 });
 
+router.post("/:name/transfer", async (req, res, next) => {
+  try{
+    var errorMessage = "";
+
+    let senderName = req.params['name'];
+    let {receiverName, amount} = req.body;
+    let sender = await User.findOne({ userName: senderName });
+    let receiver = await User.findOne({ userName : receiverName });
+
+    if(receiver != null){
+      if(sender.accountBalance > amount){
```

Comment:
Can we make it "if(sender.accountBalance >= amount)" instead?

Reply:
Changed it in latest commit